avm/ptn/authorization/policy-assignment/tests/e2e/sub.min/dependencies.bicep

```diff
@@ -0,0 +1,13 @@
+@description('Optional. The location to deploy resources to.')
```

Comment:
Is this file and its output used anywhere? Same for the other `interim` file...

Reply:
It is used to create and pass the managed identity

Reply:
I would agree if it were the case ;) Just saw you removed it. Will resolve